app/controllers/datasets/datafiles_controller.rb

```diff
@@ -1,4 +1,3 @@
-# coding: utf-8
```

Comment:
This is included because the file has UTF-8 characters. For example:

```
flash[:alert] = "Are you sure you want to delete ‘#{@datafile.name}’?"
```

Reply:
Ah, sorry - I assumed it was something someone's editor inserted.

Reply:
I think that's a correct assumption, but it would have been because of those quote characters.